docs/integrations/destinations/bigquery.md

```diff
@@ -1,2 +1,103 @@
+
+#### Service Account 
+
+In order for Airbyte to sync data into BigQuery, it needs credentials for a [Service Account](https://cloud.google.com/iam/docs/service-accounts) with the "BigQuery User" role, which grants permissions to run BigQuery jobs, write to BigQuery Datasets, and read table metadata.We highly recommend that this Service Account is exclusive to Airbyte for ease of permissioning and auditing. However, you can use a pre-existing Service Account if you already have one with the correct permissions. 
+
+The easiest way to create a Service Account is to follow GCP's guide for [Creating a Service Account](https://cloud.google.com/iam/docs/creating-managing-service-accounts). Once you've created the Service Account, make sure to keep its ID handy as you will need to reference it when granting roles. Service Account IDs typically take the form  `<account-name>@<project-name>.iam.gserviceaccount.com`  
+
+Then, add the service account as a Member in your Google Cloud Project with the "BigQuery User" role. To do this, follow the instructions for [Granting Access](https://cloud.google.com/iam/docs/granting-changing-revoking-access#granting-console) in the Google documentation. The email address of the member you are adding is the same as the Service Account ID you just created. 
+
+At this point you should have a service account with the "BigQuery User" project-level permission.
+
+#### Service Account Key 
+
+Service Account Keys are used to authenticate as Google Service Accounts. For Airbyte to leverage the permissions you granted to the Service Account in the previous step, you'll need to provide its Service Account Keys. See the [Google documentation](https://cloud.google.com/iam/docs/service-accounts#service_account_keys) for more information about Keys. 
+
+Follow the [Creating and Managing Service Account Keys](https://cloud.google.com/iam/docs/creating-managing-service-account-keys) guide to create a key. Dataline currently supports JSON Keys only, so make sure you create your key in that format. As soon as you created the key, make sure to download it, as that is the only time Google will allow you to see its contents. Once you've successfully configured BigQuery as a destination in Airbyte, delete this key from your computer. 
+
+### Setup the BigQuery destination in Airbyte
+
+You should now have all the requirements needed to configure BigQuery as a destination in the UI. You'll need the following information to configure the BigQuery destination: 
+
+* **Project ID**
+* **Dataset ID**
+* **Service Account Key**: the contents of your Service Account Key JSON file
+* **Default Target Schema:** the name of the schema where the tables will be created. In most cases, this should match the Dataset ID. 
```

Comment:
@jrhizor is this field required? the target's docs say it's not, so i'm wondering if we can just do away with it (or internally handle it by copying its value from dataset id)

Reply:
I think we should do the same thing we did for filtered db and remove it completely

Reply:
^ definitely agree